### ISSUE TITLE
Linux support?

### ISSUE THREAD

**Issue by shesek** [bittorrenttorque/btapp#11]
I would really like to play with btapp... any chance you could make Torque work on Linux?

**Comment by hemzaz** [comment]
+1

**Comment by Legogris** [comment]
+1. I don't think this has much chance of taking off before Linux is treated as a platform with at least as high priority as OS X or Windows.

**Comment by shesek** [comment]
Yes, especially if you want to attract developers to explore it and develop things around that. I really wanted to try btapp for a project I was working on, but had to give up due to the lack of Linux support.

**Comment by tyoc213** [comment]
I quite don't get it, but isn't http://btappjs.com/ pure javascript?

http://btappjs.com/getting-started.html

Or Im lost? I just want to try, havent do it.

**Comment by ulpian** [comment]
doesnt work through wine, hard to even install but even then tried running browser through wine with torque installed and did not work

**Comment by tyoc213** [comment]
I will suggest a VM with WIndows then :S??... haven't seen that https://torque.bittorrent.com/Torque.pkg is only OSX and Windows? 

My suggestion for the developers... if the original sources for the plugin are in C, then try to use https://github.com/kripken/emscripten and perhaps http socket API (maybe?... see to the future and more wide usage of those technologies if you don't want to get stuck on plugin maintenance...)... and stop trying to maintain a plugin all the way.

Or liberate the source as a permissive OSS :).

**Comment by tyoc213** [comment]
Thought I will love Linux support, I just found this: http://www.modern.ie/ they are from M$... so they are OK to use.

Shame that there is no Linux plugin.

Here are several important pieces of information about free Windows VMs. As you can see, the free Windows VMs are for evaluation purposes, and are not supposed to be used for any production system.

```
By default, the imported Windows VM has its network adapter not attached to any network. So, open up network settings of the WinXP VM, and make the network adapter attached to bridged network.
The Windows VM has a set of basic Windows utilities installed, as well as VirtualBox Guest Additions for Windows.
You may use the free Windows VM for testing purposes only, but not for commercial purposes.
You can use the free Windows VM only for 90 days after it is installed. After 90 days, the Windows VM will stop running, and you will not be able to access data used in the VM.
Windows 8.1 Preview version expires on 1/15/2014.
Login information for Windows Vista, 7 and 8 VMs: IEUser, Passw0rd!
```

http://xmodulo.com/2013/07/how-to-install-and-run-microsoft-windows-for-free-on-linux.html

**Comment by tyoc213** [comment]
And google and FF will be pushing to eliminate plugins :P... (this is just starting)
- http://blog.chromium.org/2013/09/saying-goodbye-to-our-old-friend-npapi.html
- https://blog.mozilla.org/futurereleases/2013/09/24/plugin-activation-in-firefox/
- https://blog.mozilla.org/security/2013/01/29/putting-users-in-control-of-plugins/

So you can go to future and support HTML5 (there is a option up there for C on javascript) and in that point... you will support Linux, OSX and Windows.... and others that are able to HTML5.

**Comment by FundCoders** [comment]
Looking forward to this enhancement? Bid some money, get a quicker implementation and reward the coders on [FundCoders.com](http://www.fundcoders.com/Bid#/bittorrenttorque/btapp/issues/11)
